code/common/pubsub.q

```diff
@@ -187,4 +193,4 @@ init:{[t]
     val:.ds.subreq input;
   ];
   skey=val
- };
```

Comment:
Was this deleting a spare line at the end of the file?

Reply:
I believe this was adding a new line to remove this warning in git diffs.